string roboter_name
int32 machine_id
bool got_r
string resource_name
bool resource_blocked
string object_name
int32 id
